src/components/_carousel.scss

```diff
@@ -229,7 +229,9 @@
 		bottom: 0;
 		margin: 10px 0;
 		text-align: center;
-		width: 100%;
+		width: auto;
```

Comment:
You can apply this css rule locally as an override. I think that would be safer than changing these styles here for one single use case.